src/VisualStudio/Core/Def/Implementation/Remote/JsonRpcConnection.cs

```diff
@@ -16,68 +16,56 @@ namespace Microsoft.VisualStudio.LanguageServices.Remote
 {
     internal class JsonRpcConnection : RemoteHostClient.Connection
```

Comment:
potentially doc these tyeps.  i get very confused as to what the purpose is of all this scaffolding code

Reply:
In progress. I am not sure either what some of these are really for. More refactoring is coming...

Reply:
👍

Reply:
FYI: most of my feedback is in teh category of: as you continue to work in this space, improving this stuff owuld be great.  i don't expect it all to change in this PR